### PR TITLE
fix(docs): Added extra information for step 8 to avoid confusion on r…

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard30**
- Added the extra information to avoid confusion with the Release draft target branch

### CODE REVIEW

File: docs/release-procedure.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
 7. Once ready, merge the release branch to `master` using a **merge commit.
    Never use a squash and merge action, since this will erase/rewrite the commit
    history.**
-8. Hit the publish button in the GitHub release draft. This should tag the
+8. After completing step 7, change the target of the release draft to `master` and hit the publish button in the GitHub release draft. This should tag the
```
Comment (VWSCoronaDashboard18) [inline docs/release-procedure.md:24]: Now the last part feels a bit unnecessary. You can remove "this should tag the correct commit in `master`"

**Review by VWSCoronaDashboard18 — APPROVED** [review]
This makes it more fail proof! I like it.

File: docs/release-procedure.md (hunk 1 above)
Comment (VWSCoronaDashboard30) [reply docs/release-procedure.md]: Done!

**Review by VWSCoronaDashboard18 — DISMISSED** [review]
Good stuff! Nice and clear!

File: docs/release-procedure.md (hunk 1 above)
Comment (VWSCoronaDashboard28) [reply docs/release-procedure.md]: Im not so sure about removing the last part as it adds the "Why" for this step.

**Review by VWSCoronaDashboard28 — APPROVED** [review]
Approved but please see me response to the comment thread started by @VWSCoronaDashboard18

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Review by VWSCoronaDashboard18 — APPROVED** [review]
LG2M

**Pull request merged**